src/universal/components/SignInPage/SignInPage.js

```diff
@@ -116,29 +85,24 @@ class SignInPage extends Component<Props, State> {
     this.setState({loggingIn: false, error: null});
   };
 
-  saveToken = (response: ParsedAuthResponse): void => {
+  saveToken = (response: AuthResponse): void => {
```

Comment:
naming-- `saveToken` is only half of what `signinAndUpdateToken` actually does. maybe dethunkify & call that function directly?

Reply:
I'll rename this to `signIn`, but keep it, since the rest of the parameters of `signinAndUpdateToken` are bound to the component instance, rather than dynamic with respect to the signing-in action.